src/components/Sidebar.jsx

```diff
@@ -484,22 +492,22 @@ const Sidebar = ({
                       </FormFieldWrapper>
                     )}
 
-                    {['select', 'radio', 'checkbox'].indexOf(
+                    {['select', 'single_choice', 'multiple_choice'].indexOf(
                       subblock.field_type,
                     ) >= 0 && (
-                      <ArrayWidget
-                        id="input_values"
-                        title={intl.formatMessage(messages.field_input_values)}
-                        onChange={(name, value) => {
-                          onChangeSubBlock(index, {
-                            ...subblock,
-                            [name]: value,
-                          });
-                        }}
-                        required={true}
-                        value={subblock.input_values}
-                      />
-                    )}
+                        <ArrayWidget
+                          id="input_values"
+                          title={intl.formatMessage(messages.field_input_values)}
+                          onChange={(name, value) => {
+                            onChangeSubBlock(index, {
+                              ...subblock,
+                              [name]: value,
+                            });
+                          }}
+                          required={true}
+                          value={subblock.input_values}
+                        />
+                      )}
```

Comment:
Not sure if this should be indented like this.. was it checked by prettier? Just double checking to be 100% sure.